data/gallery/balustrades.ts

```diff
@@ -0,0 +1,22 @@
+export const images = [
+  { src: "/gallery/balustrady/1.jpg", title: "1" },
```

Comment:
we also need to figure out how to display independently the image number (title) and `alt`